<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>341</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Мотопарк</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Terminus</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Мотопарк</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnPrintAll">
        <property name="text">
         <string>Все мотоциклы и владельцы</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/5b19c51e1cd2c684cf9a401053aa2dbe-Photoroom.png-Photoroom.png</normaloff>../../Загрузки/5b19c51e1cd2c684cf9a401053aa2dbe-Photoroom.png-Photoroom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/1828819.png</normaloff>../../Загрузки/1828819.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/3946884.png</normaloff>../../Загрузки/3946884.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Поиск</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/png-transparent-magnifying-glasses-illustration-magnifying-glass-icon-magnifier-cartoon-glasses-magnifier-thumbnail(1)-Photoroom.png-Photoroom.png</normaloff>../../Загрузки/png-transparent-magnifying-glasses-illustration-magnifying-glass-icon-magnifier-cartoon-glasses-magnifier-thumbnail(1)-Photoroom.png-Photoroom.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStatictics">
        <property name="text">
         <string>Узнать статистику</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/3721812.png</normaloff>../../Загрузки/3721812.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/Rettungszeichen_Notausgang_Links_(alt).svg.png</normaloff>../../Загрузки/Rettungszeichen_Notausgang_Links_(alt).svg.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>111</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>110</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
